x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/BlockUtils.java

```diff
@@ -214,6 +219,16 @@ public static void appendValue(Block.Builder builder, Object val, ElementType ty
         }
     }
 
+    private static BytesRef toBytesRef(Object val) {
+        if (val instanceof SpatialPoint point) {
+            return new BytesRef(WellKnownBinary.toWKB(new Point(point.getX(), point.getY()), ByteOrder.LITTLE_ENDIAN));
+        }
+        if (val instanceof Geometry geometry) {
+            return new BytesRef(WellKnownBinary.toWKB(geometry, ByteOrder.LITTLE_ENDIAN));
+        }
```

Comment:
I just run the tests and these lines above seem to never be called, can we remove it? Otherwise could you point me where this code is exercise?

Reply:
Note that I think we should never have those classes at this point, so I think this is not necessary.

Reply:
This is now removed